app/screens/participants_list/participant_row/index.tsx

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {injectIntl, intlShape} from 'react-intl';
+import {Text, TouchableOpacity, View} from 'react-native';
+
+import {showModal} from '@actions/navigation';
+import CompassIcon from '@components/compass_icon';
+import ProfilePicture from '@components/profile_picture';
+import type {Theme} from '@mm-redux/types/preferences';
+import {UserProfile} from '@mm-redux/types/users';
+import {displayUsername} from '@mm-redux/utils/user_utils';
+import {preventDoubleTap} from '@utils/tap';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {Preferences} from '@mm-redux/constants';
+
+interface ParticipantRowProps {
+        theme: Theme,
+        user: UserProfile,
+        intl: typeof intlShape;
+    }
+
+const ParticipantRow = ({theme, user, intl}: ParticipantRowProps) => {
+    const goToUserProfile = async () => {
+        const {formatMessage} = intl;
+        const screen = 'UserProfile';
+        const title = formatMessage({id: 'mobile.routes.user_profile', defaultMessage: 'Profile'});
+        const passProps = {
+            userId: user.id,
+        };
+
+        const closeButton = await CompassIcon.getImageSource('close', 24, theme.sidebarHeaderTextColor);
+
+        const options = {
+            topBar: {
+                leftButtons: [{
+                    id: 'close-settings',
+                    icon: closeButton,
+                }],
+            },
+        };
+
+        showModal(screen, title, passProps, options);
+    };
+
+    if (!user.id) {
+        return null;
+    }
+
+    const {id, username} = user;
+    const usernameDisplay = '@' + username;
+    const displayName = displayUsername(user, Preferences.DISPLAY_PREFER_FULL_NAME);
```

Comment:
is it always Prefer full name? why is this not respecting the user setting for display names ?